ts/features/zendesk/components/ZendeskSupportComponent.tsx

```diff
@@ -91,7 +104,8 @@ const ZendeskSupportComponent = (props: Props) => {
       .getOrElse({});
 
     setUserIdentity(zendeskIdentity);
-  }, [zendeskToken, profile]);
+    requestTicketNumber();
+  }, [requestTicketNumber, zendeskConfig, zendeskToken, profile]);
```

Comment:
There is an ignored warning 
![Schermata 2021-12-22 alle 09 45 07](https://user-images.githubusercontent.com/822471/147063418-0b36088c-81e7-4639-97f1-0761718e2cfe.png)

that causes this

https://user-images.githubusercontent.com/822471/147063438-686561cc-c304-47aa-9367-d9fd61698611.mov

this suggestion should fix
```suggestion
    dispatch(zendeskRequestTicketNumber.request());
  }, [dispatch, zendeskConfig, zendeskToken, profile]);
```

Reply:
Thank you! Modified in [462dab3](https://github.com/pagopa/io-app/pull/3605/commits/462dab39bd42e373b5fd3ef41a18a7baa3abf6ab)